packages/lesswrong/components/recentDiscussion/RecentDiscussionSubforumThread.tsx

```diff
@@ -0,0 +1,151 @@
+import React, {useState, useCallback} from 'react';
+import { Components, registerComponent, } from '../../lib/vulcan-lib';
+import withErrorBoundary from '../common/withErrorBoundary'
+import { tagGetSubforumUrl } from '../../lib/collections/tags/helpers';
+import { Link } from '../../lib/reactRouterWrapper';
+import { useRecordSubforumView } from '../hooks/useRecordSubforumView';
+import { useCurrentUser } from '../common/withUser';
+
+const styles = (theme: ThemeType): JssStyles => ({
+  root: {
+    marginBottom: theme.spacing.unit*4,
+    position: "relative",
+    minHeight: 58,
+    boxShadow: theme.palette.boxShadow.default,
+    borderRadius: 3,
+    backgroundColor: theme.palette.panelBackground.recentDiscussionThread,
+  },
+  subforumTitleRow: {
+    display: 'flex',
+    columnGap: 7,
+    marginBottom: 12,
+    '& svg': {
+      height: 20,
+      width: 20,
+      fill: theme.palette.grey[700]
+    }
+  },
+  subforumIcon: {
+    marginTop: 2
+  },
+  subforumTitleText: {
+    display: 'flex',
+    flexWrap: 'wrap',
+    alignItems: 'baseline',
+    columnGap: 12
+  },
+  subforumTitle: {
+    color: theme.palette.grey[900],
+    fontFamily: theme.typography.postStyle.fontFamily,
+    fontSize: 20,
+    lineHeight: '25px',
+  },
+  subforumSubtitle: {
+    color: theme.palette.grey[600],
+    fontFamily: theme.typography.fontFamily,
+    fontSize: 13,
+    lineHeight: '18px',
+    marginTop: 4,
+    marginLeft: 1
+  },
+  tag: {
+    paddingTop: 18,
+    paddingLeft: 16,
+    paddingRight: 16,
+    background: theme.palette.panelBackground.default,
+    borderRadius: 3,
+    marginBottom: 4
+  },
+  content: {
+    padding: '0 4px 4px 16px',
+    [theme.breakpoints.down('sm')]: {
+      padding: '0 4px 1px 4px',
+    }
+  },
+});
+
+const RecentDiscussionSubforumThread = ({ comment, tag, refetch = () => {}, expandAllThreads: initialExpandAllThreads, classes }: {
+  comment: CommentWithRepliesFragment,
+  tag: TagBasicInfo|null,
+  refetch?: any,
+  expandAllThreads?: boolean
+  classes: ClassesType
+}) => {
+  // const currentUser = useCurrentUser()
+  // const recordSubforumView = useRecordSubforumView({userId: currentUser?._id, tagId: tag?._id})
+
+  const [expandAllThreads, setExpandAllThreads] = useState(false)
+  // const [markedAsVisitedAt, setMarkedAsVisitedAt] = useState<Date|null>(null)
+
+  // TODO: do we need to default this to be the last time this user viewed the subforum?
+  //       also, do we want to use the "highlightDate" option?
+  // const lastVisitedAt = markedAsVisitedAt
+
+  // TODO: do we want to count this as a subforum view?
+  // const markAsRead = useCallback(
+  //   () => {
+  //     setMarkedAsVisitedAt(new Date())
+  //     setExpandAllThreads(true)
+  //     recordSubforumView()
+  //   },
+  //   [recordSubforumView]
+  // )
```

Comment:
I left some TODOs in here, because I'm not sure what the right behavior is, but feel like it's not needed for the MVP so we can figure it out in a later PR. I'm not entirely sure what `markAsRead` and `highlightDate` in `treeOptions` are supposed to do - I think someone should look into this later and decide whether we want to use them here.